frontend/src/views/Collection/components/DatasetUploadToast/index.tsx

```diff
@@ -1,7 +1,17 @@
-import { Position, Toaster } from "@blueprintjs/core";
+import { IToaster, IToastProps, Position, Toaster } from "@blueprintjs/core";
 
-const DatasetUploadToast = Toaster.create({
```

Comment:
`.create()` actually queries and inserts a new element in the DOM, which doesn't work for SSR. So I changed this to only call `.create()` when `.show()` is called on the client side